### PR TITLE
Add Functionality for Viewing Last Entry Time

### PR DESCRIPTION
**Pull request by dinobi**
### Feature
* This PR adds functionality that allows a user see the last time they created an entry and links them to that particular entry

### Screenshot
screen shot 2019-02-12 at 1 58 48 am

### CODE REVIEW

File: app/presenters/base_presenter.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class BasePresenter
+  attr_reader :template
+
+  def initialize(model, template)
+    @model = model
+    @template = template
+  end
+
+  def last_entry_time
+    "Your last entry was #{template.time_ago_in_words(@model.created_at)} ago"
```
Comment (fatboypunk) [inline app/presenters/base_presenter.rb]: I think we want this in i18n

File: config/locales/pt-BR.yml [hunk 2]
```diff
@@ -217,6 +217,8 @@ pt-BR:
     no_projects_html: Parece que você ainda não tem nenhum projeto definido, você pode definir um em %{new_project_path}.
     no_billable_projects_html: Parece que você ainda não tem nenhum projeto faturável, você pode definir um em %{new_project_path}.
     no_billable_entries_html: Parece que nenhuma entrada preenche os requisitos de filtro.
+    no_entries: You have not created any entries
```
Comment (jattoabdul) [inline config/locales/pt-BR.yml]: Shouldn't this be written in the language for this locale `pt-BR.yml`?

File: config/locales/nl.yml [hunk 3]
```diff
@@ -223,6 +223,8 @@ nl:
       %{new_project_path} doen.
     no_billable_projects_html: Zo te zien heb je nog geen factureerbare projecten. Dat kun je %{new_project_path} doen.
     no_billable_entries_html: Zo te zien heb je geen entries die voldoen aan je zoekcriteria.
+    no_entries: You have not created any entries
```
Comment (jattoabdul) [inline config/locales/nl.yml]: Shouldn't this be written in the language for this locale `nl.yml`?

**Comment by jurre** [comment]
We've decided to freeze feature development as none of the maintainers currently have time to properly steward contributions from the community.

**Pull request closed without merging**